### ISSUE TITLE
Enhancement: Load subtitles from the current .mkv file

### ISSUE THREAD

**Issue by milichev** [tcrespog/vlc-subtitle-word-search#8]
## Current state

Currently, one needs to find subtitles via `VLsub`, and load, if any are found.

## Problems

1. The media file might already contain embedded subtitles, therefore it is not needed to find ones elsewhere.
2. On top of that, the embedded subtitles might be uniquely prepared, synced, etc.
3. When the current media file changes, the selected subtitles in the extension remain the same. 

## Proposed functionality

1. When the current media file contains subtitles (f.x. a *.mkv file), the extension should load ones.
2. When the media file changes, the extension has to select subtitles appropriately, either embedded ones or previously downloaded. The embedded subtitles have precedence. Apparently, the extension should listen to an event of file change.

**Comment by NemanSyed** [comment]
Agreed. I have many MKVs where the subtitles are embedded so the end user only has one file to manage without regard to how many languages. It would be immensely helpful if this tool could optionally search the currently selected embedded subtitles, Thank you!
